src/http/body.rs

```diff
@@ -50,3 +57,24 @@ impl<T: IntoRequestBody> IntoRequestBody for Option<T> {
         }
     }
 }
+
```

Comment:
We need one of these for each `impl IntoRequestBody for` like the example below for `Option<T>`.

```rust
#[cfg(test)]
mod test_option {
  use crate::prelude::*;
  use http::Request;
  #[test]
  fn test_send() {
      Request::post("https://example.com").body(()).unwrap().send(Some(&[4u8]));
      Request::post("https://example.com").body(Some(&[4u8])).unwrap().send();
  }
}
```